coderabbit.yml

```
@@ -107,14 +107,117 @@ reviews:
     - path: "src/**/*.{ts,tsx}"
       instructions: |
-        Focus on TypeScript/React code quality:
-        - Ensure proper type safety and avoid 'any' types
-        - Check for proper error handling and async/await patterns
-        - Verify React hooks usage and component patterns
-        - Review for security vulnerabilities and performance issues
-        - Ensure proper JSDoc comments for functions and complex logic
-        - Check for proper imports and exports
-        - Verify proper state management patterns
+        Focus on TypeScript/React code quality aligned with Biome configuration:
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)